.github/workflows/ci.yml

```diff
@@ -79,16 +79,16 @@ jobs:
           path: screencap.png
 
       - name: create unit coverage
-        if: ${{ matrix.api-level==21 }}
+        if: ${{ matrix.api-level==24 }}
```

Comment:
@gouri-panda Last time we placed this condition as @kelson42 requested, `Unit coverage` works similarly for every API level and we are testing all the UI functionality in `instrumentation test cases` for every API level, so is it beneficial to run `unit coverage` for every API level?

Reply:
its would be too much for code coverage only.

Reply:
@MohitMaliDeveloper but we are here running this `run: ./gradlew testDebugUnitTest testCustomexampleDebugUnitTest`

Reply:
@gouri-panda We are running unit tests for both the custom and app modules i think this naming is good since it creates the coverage for the whole project, if you have a better suggestion for this please share your thoughts.